config/analytics.yml

```diff
@@ -4,9 +4,3 @@
 analytics:
   ga4:
     analytics_id: <%= ENV['GOOGLE_ANALYTICS_ID'] %>
-    app_name: <%= ENV.fetch('GOOGLE_OAUTH_APP_NAME', '') %>
```

Comment:
These need to remain.  They are so that the proprietor can fetch data from Google Analytics (and later, when working, render that information in Hyrax).

Reply:
do you mean to restore just this line that you are pointing to or the whole thing

Reply:
You need to restore all of the keys that were in this file.  These values provide the ability to *fetch* data from Google Analytics.